packages/design/src/DataTable/Cells.tsx

```diff
@@ -102,13 +106,12 @@ export const ClickableLabelCell = ({
       onClick={() => onClick(label)}
       key={`${label.name}:${label.value}`}
       mr="1"
-      mb="1"
       kind="secondary"
       css={`
         cursor: pointer;
       `}
     >
-      {`${label.name}: ${label.value}`}
+      <div css={{ lineHeight: '20px' }}>{`${label.name}: ${label.value}`}</div>
```

Comment:
this applies line height to each label, which won't have the affect we want. we need this style wrapping the entire labels 

```
return <Cell><div css={{ lineHeight: '20px' }}>{$labels}</div></Cell>;
```

Reply:
bump. to see this in storybook to visually test it, is to look at resource lists like `nodes`, `apps`, `desktops`, etc.